### PR TITLE
Notes 20200204

### PR DESCRIPTION
**Pull request by hawnzug**

### CODE REVIEW

File: notes-20200204.tex [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+% \begin{tabular}{c|p{75}|c|c}
+%     Classification & Cat Theory & Sequence Calculus & Type Theory\\\hline
+%     Negative & mapping in universal property & right rules invertible & $\Pi$ $\Sigma$ $\pairtype$ $\funtype$ $\top$\\\hline
+%     Positive & mapping out universal property & left rules invertible & $\sumtype$ $\bot$\\\hline
+% \end{tabular}   
+%\end{center}
+
+\begin{center}
+ \begin{tabular}{c|c|c|c}
+     Classification & Cat Theory & Sequence Calculus & Type Theory\\\hline
```
Comment (jonsterling) [inline notes-20200204.tex]: sequent :wink:

Comment (hawnzug) [reply notes-20200204.tex]: :sweat_smile: thanks!

File: notes-20200204.tex [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+ %\begin{tabular}{@{}c@{}}Three \\ Drie\end{tabular}
+Difference between negative and positive in type theory:
+\begin{enumerate}
+    \item Negative: The introduction rule is constructed from the elimination rule.
+    \item Positive: The elimination rule is constructed from the introduction rule.
+\end{enumerate}
+However, we know there are two possible ways to define the pair type. The first one is using the two projections we present here. The other way is using the $\mathtt{split}$ operator mentioned in the previous lecture. If we define the product type using the second way, its polarity then becomes positive.
+
+\section{Martin-L\"of ethos/mythos}
+Every "cool" type is an internalization of some judgmental structure.
```
Comment (favonia) [inline notes-20200204.tex]: You should use ` `` ` and `''`.

File: notes-20200204.tex [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    \hypo{\oftype{M}{\dpairtype{x}{A}{B}}}
+    \infer1{\eqterm{\pair{\pairfst{M}}{\pairsnd{M}}}{M}{\dpairtype{x}{A}{B}}}
+  \end{prooftree*}
+  Notice that the elimination rules tell us
+  $\oftype{\pairfst{M}}{A}$ and $\oftype{\pairsnd{M}}{B\subst{\pairfst{M}}{x}}$.
+  Then by the introduction rule we know
+  $\oftype{\pair{\pairfst{M}}{\pairsnd{M}}}{\dpairtype{x}{A}{B}}$.
+\end{enumerate}
+
+\section{Dependent Sum Types}
```
Comment (favonia) [inline notes-20200204.tex]: I probably made a mistake. This should be "disjoint sum types".

Comment (hawnzug) [reply notes-20200204.tex]: Oops it's my fault. I added "disjoint" in the first section but forgot to change this one.

**Review by favonia — APPROVED** [review]
Looks fine to me! Nice work.

**Pull request merged**